examples/chart/teleport-cluster/values.yaml

```diff
@@ -28,6 +28,33 @@ acmeEmail: ""
 # acmeURI is the ACME server to use for getting certificates. The default is to use Let's Encrypt's production server.
 acmeURI: ""
 
+service:
+  type: "LoadBalancer"
+  load_balancer_ip: ""
+  https_port: "443"
+  sshproxy_port: "3023"
+  k8s_port: "3026"
+  sshtun_port: "3024"
+
+ingress:
+  enabled: false
+  # Used to create Ingress record (should used with service.type: ClusterIP).
+  hosts:
+    - ""
+  # - teleport.domain.com
+  # The ingress path. Useful to host teleport on a subpath, such as `/teleport`.
```

Comment:
Is this tested functionality? I'm not sure that tsh would be able to find teleport if it's anywhere other than `/`.

Reply:
If you define the complete path as a proxy in tsh it shouldn't have any issue.

Reply:
Teleport doesn't support the usage of a subpath for `tsh` or other tooling - the service must be available at the root of the proxy URL.